ext/afform/core/ang/af/afForm.component.js

```diff
@@ -137,11 +137,23 @@
         return str;
       }
 
+      function validateFileFields() {
+        var valid = true;
+        $("#bootstrap-theme input[type='file']").each((index, fld) => {
```

Comment:
@kurund this all looks good, except the jQuery selector of the form is not specific enough. It needs to target the individual form being validated.

Reply:
I have made the selector more specific to the form. Let me know if that works!